components/HomeScreen.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from "./HomeScreen.module.css";
+import Articles from './sub-components/Articles';
+import NavBar from "./sub-components/NavBar";
+import Header from "./sub-components/Header";
+
+const HomeScreen = ({setVotes, loading, setLoading}) => {
+    return <div className={styles.grid_container}>
+        <div className={styles.header}><Header /></div>
+        <div className={styles.navbar}><NavBar /></div>
+        <div className={styles.articles}><Articles setVotes={setVotes} loading={loading} setLoading={setLoading}/></div>
+        </div>
```

Comment:
There are a lot of `<div>` tags wrapping your React components, it might be necessary for the styling effect you want, but you should investigate whether it's possible to cut down on `<div>` tags where you can 🧹